izanami-frontend/src/pages/tenantSettings.tsx

```diff
@@ -114,9 +118,14 @@ export function TenantSettings(props: { tenant: string }) {
                 .then(() => setInviting(false))
             }
             cancel={() => setInviting(false)}
+            invitedUsers={usersData?.map((user) => user.username) || []}
           />
         )}
-        <TenantUsers tenant={tenant} />
+        <TenantUsers
+          tenant={tenant}
+          isLoading={isLoading}
```

Comment:
Same as for Project (and others), TenantUsers should not be responsible for displaying loading indicator.